projects/sw-offline-google-analytics/README.md

```diff
@@ -15,20 +15,24 @@ Browse sample source code in the [demo directory](https://github.com/GoogleChrom
 
 ### goog.useOfflineGoogleAnalytics
 
-[projects/sw-offline-google-analytics/src/offline-google-analytics-import.js:182-229](https://github.com/GoogleChrome/sw-helpers/blob/213765485d2c3378497e2a80c2b80993a7b09cdf/projects/sw-offline-google-analytics/src/offline-google-analytics-import.js#L182-L229 "Source code on GitHub")
+[projects/sw-offline-google-analytics/src/offline-google-analytics-import.js:45-83](https://github.com/GoogleChrome/sw-helpers/blob/5320c8269f3368939bf792c4bc1a47487ca7963f/projects/sw-offline-google-analytics/src/offline-google-analytics-import.js#L45-L83 "Source code on GitHub")
 
-`goog.useOfflineGoogleAnalytics` is the main entry point to the library
-from within service worker code.
+In order to use the library, call`goog.useOfflineGoogleAnalytics()`.
```

Comment:
This matches what's done with `sw-appcache-behavior`, which exposes [`goog.legacyAppCacheBehavior()`](https://www.npmjs.com/package/sw-appcache-behavior#googlegacyappcachebehavior).

I'm open to changing both, and renaming the method exposed in `sw-appcache-behavior` to `goog.appCacheBehavior.legacy()` (or something like that)namespace if you think that's preferable.

Being consistent across projects is my main concern.